tox.ini

```diff
@@ -114,6 +115,30 @@ commands =
     rm -rf {envdir}/.pytest_cache
     make regen
 
+[testenv:plugins]
```

Comment:
how about?

```suggestion
[testenv:plugins]
pip_pre=true
```
https://tox.readthedocs.io/en/latest/config.html#conf-pip_pre

Reply:
To test pre-release of plugins you mean? I don't see many plugins doing that (never seen any TBH). You think it is worth it?

Reply:
What's the cost that could make it not worth it?  I don't pre-release in most cases but I feel like the world might be better if we all did and we call checked against those.  Maybe.

Reply:
Yeah I think it's worth it, would encourage pre-releases even if none are made now

Reply:
Fair enough, thanks! Done.